.github/workflows/Pkg EdFi.Ods.Minimal.Template.TPDM.yml

```diff
@@ -54,8 +72,9 @@ jobs:
       run: | 
         echo "::error::Missing Azure Token"
         exit 1 
-    - name: Support longpaths
-      run: git config --system core.longpaths true
+    - name: Set permissions for database server container backup directory
+      run: docker exec --user root mssql chmod 777 /DatabaseBackups
+      shell: bash
```

Comment:
Can you change it to pwsh instead of bash ?

Reply:
@semalaiappan , can you elaborate on the reasoning behind the changes from bash to pwsh? Bash is the default shell on Ubuntu, and the steps where this change is requested are all intended to execute a binary on the Ubuntu host. The "Set permissions for database server container backup directory" steps directly invoke the docker executable on the runner host, and the "Set permissions for database backup file" steps execute a Linux-specific command (chmod), which modifies permissions on the Ubuntu host's file system in a way without a direct equivalent in windows or PowerShell. I don't believe the change to pwsh would cause any harm. However, I hesitate to use pwsh if its unique capabilities are not required. It appears that for these steps, it would only act as an additional layer that the one-line commands must pass through to the run in the underlying Ubuntu environment.